stylesheets/_typography.scss

```diff
@@ -40,13 +40,13 @@ $is-print: false !default;
   font-weight: $font-weight;
```

Comment:
The line-height isn't causing that much trouble. But you're right, it should be moved into the else case, together with the other font-size. And the normal font-size needs to be copied into the print style. I will adjust the commit accordingly.
I'm not sure I understand what you mean with "Only conditionally having the media query feels arbitrary". I would generally never have `@media` queries inside a print stylesheet. (Although I can think of legitimate reasons, I've never seen it used before.)